### PR TITLE
Composite Sounding and map reduce Check-in

### PR DESCRIPTION
**Pull request by dkimpara**
added to env.yml also

### CODE REVIEW

**Comment by ThomasMGeo** [comment]
Fan of Black eh :)

**Comment by dkimpara** [comment]
@djgagne, @jsschreck asked me to tag you

File: composite-soundings/soundings/utils.py
Comment (ThomasMGeo) [inline composite-soundings/soundings/utils.py:1]: any reason this is in utils and not plotting?

**Comment by dkimpara** [comment]
copied results for all map reduce computations for composite soundings to `/glade/campaign/cisl/aiml/ptype/composite_stats_ptype_case_studies`

**Comment by djgagne** [comment]
I fixed the one conflict found and did not see any other potential conflicts with existing code, so I am going to merge now.

**Pull request merged**